docs/content/document/add-documentation.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: "Add documentation"
-weight: 50
+weight: 10
+aliases:
+  - /docs/develop/add-documentation
```

Comment:
The `/docs/` prefix is on some older aliases because there used to be a `docs` folder inside `docs/content` but that's no longer the case.

```suggestion
  - /develop/add-documentation
```